### PR TITLE
Ehcache monitor broken out to separate module

### PR DESCRIPTION
**Pull request by mmoayyed**

### CODE REVIEW

File: cas-server-integration-ehcache-monitor/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+ *   <li>Monitoring : follow separately the number of TGT and ST.</li>
+ * </ul>
+ *
+ * @author <a href="mailto:cleclerc@xebia.fr">Cyrille Le Clerc</a>
+ * @author Adam Rybicki
+ * @author Andrew Tillinghast
+ * @since 3.5
+ */
+@Component("ehcacheTicketRegistry")
+public final class EhCacheTicketRegistry extends AbstractCrypticTicketRegistry implements InitializingBean {
```
Comment (leleuj) [inline cas-server-integration-ehcache-monitor/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java]: Why is the ticket registry in the monitor module?

Comment (mmoayyed) [inline cas-server-integration-ehcache-monitor/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java]: Good catch. leftover from the move, and should be removed

Comment (mmoayyed) [inline cas-server-integration-ehcache-monitor/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java]: Fixed and updated the pull.

**Comment by leleuj** [comment]
Why no file is removed from the current `cas-server-integration-ehcache` module and moved to the new monitor one?

**Comment by mmoayyed** [comment]
There should be. When I review the diff, I see:

```
 ...org/jasig/cas/monitor/EhCacheMonitor.java → ...org/jasig/cas/monitor/EhCacheMonitor.java
 ...e → cas-server-integration-ehcache-monitor}/src/main/java/org/jasig/cas/monitor/EhCacheStatistics.java
```

and so on.

**Comment by leleuj** [comment]
You're right: files are moved, not deleted.

+1

**Pull request merged**